### ISSUE TITLE
tunein: extraction error with specific station URL

### ISSUE THREAD

**Issue by bubbleguuum** [rg3/youtube-dl#7348]
./youtube-dl http://tun.in/tg7zSI -J --flat-playlist --no-check-certificate --socket-timeout 30 --verbose
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'http://tun.in/tg7zSI', u'-J', u'--flat-playlist', u'--no-check-certificate', u'--socket-timeout', u'30', u'--verbose']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.11.02
[debug] Python version 2.7.3 - Linux-3.2.0-70-generic-x86_64-with-Ubuntu-12.04-precise
[debug] exe versions: avconv 0.10.12-7, avprobe 0.10.12-7, ffmpeg 0.10.12-7, ffprobe 0.10.12-7
[debug] Proxy map: {}
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "./youtube-dl/__main__.py", line 19, in <module>
  File "./youtube-dl/youtube_dl/__init__.py", line 410, in main
  File "./youtube-dl/youtube_dl/__init__.py", line 400, in _real_main
  File "./youtube-dl/youtube_dl/YoutubeDL.py", line 1666, in download
  File "./youtube-dl/youtube_dl/YoutubeDL.py", line 661, in extract_info
  File "./youtube-dl/youtube_dl/extractor/common.py", line 290, in extract
  File "./youtube-dl/youtube_dl/extractor/tunein.py", line 58, in _real_extract
AttributeError: 'NoneType' object has no attribute 'group'

**Comment by sandalle** [comment]
Some of the videos in [this playlist](https://www.youtube.com/playlist?list=PLB5562D5DBA49DCCE) download fine, but others (such as #2 and #5) do not.

```
$ youtube-dl --no-check-certificate --verbose -cAt https://www.youtube.com/playlist?list=PLB5562D5DBA49DCCE
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'--no-check-certificate', u'--verbose', u'-cAt', u'https://www.youtube.com/playlist?list=PLB5562D5DBA49DCCE']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.11.21
[debug] Python version 2.7.10 - Darwin-15.0.0-x86_64-i386-64bit
[debug] exe versions: none
[debug] Proxy map: {}
[youtube:playlist] PLB5562D5DBA49DCCE: Downloading webpage
[download] Downloading playlist: Minecraft Songs & Parodies!
[youtube:playlist] playlist Minecraft Songs & Parodies!: Downloading 33 videos
[download] Downloading video 1 of 33
[youtube] EE-P4pmp3vk: Downloading webpage
[youtube] EE-P4pmp3vk: Downloading video info webpage
[youtube] EE-P4pmp3vk: Extracting video information
[youtube] EE-P4pmp3vk: Downloading DASH manifest
[youtube] EE-P4pmp3vk: Downloading DASH manifest
[debug] Invoking downloader on u'https://r5---sn-nx5e6n7z.googlevideo.com/videoplayback?ipbits=0&upn=gqzJ3xr1x7s&ratebypass=yes&dur=86.378&initcwndbps=731250&source=youtube&mm=31&fexp=9407053%2C9408206%2C9408501%2C9408710%2C9416126%2C9417260%2C9417683%2C9418185%2C9419452%2C9420452%2C9420539%2C9421354%2C9422596%2C9422618%2C9423244%2C9423292%2C9423429%2C9423492%2C9423662%2C9423786%2C9424963&nh=IgpwcjAyLnNlYTAzKgkxMjcuMC4wLjE&expire=1448618655&requiressl=yes&signature=72415CFBB74A0FC71511E8BF56B6FBD18CA744C8.B90EC8FDD8986674983AEB7FB582ECA37753FF21&itag=22&sver=3&lmt=1427588763722953&pl=18&mime=video%2Fmp4&id=o-AIJAua3ic_kJbJOVCRRUbKrOSJf7BO5H1Sd83JnxPNFn&mv=m&sparams=dur%2Cid%2Cinitcwndbps%2Cip%2Cipbits%2Citag%2Clmt%2Cmime%2Cmm%2Cmn%2Cms%2Cmv%2Cnh%2Cpl%2Cratebypass%2Crequiressl%2Csource%2Cupn%2Cexpire&mt=1448596954&ms=au&mn=sn-nx5e6n7z&key=yt6&ip=97.114.76.242'
[download] 00001-♫ 'THE SQUID' - Minecraft Parody of Ylvis - The Fox (ft. MlgHwnT, GizzyGazza, GoldSolace & Kuledud3)-EE-P4pmp3vk.mp4 has already been downloaded
[download] 100% of 28.08MiB
[download] Downloading video 2 of 33
[youtube] ZgpbGbrLV_M: Downloading webpage
[youtube] ZgpbGbrLV_M: Downloading video info webpage
[youtube] ZgpbGbrLV_M: Extracting video information
[youtube] {22} signature length 49.43, html5 player en_US-vflFEzfy7
[youtube] ZgpbGbrLV_M: Downloading player https://s.ytimg.com/yts/jsbin/player-en_US-vflFEzfy7/base.js
ERROR: Signature extraction failed: Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 828, in _decrypt_signature
    video_id, player_url, s
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 755, in _extract_signature_function
    cache_res = res(test_string)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 807, in <lambda>
    return lambda s: initial_function([s])
  File "/usr/local/bin/youtube-dl/youtube_dl/jsinterp.py", line 254, in resf
    res, abort = self.interpret_statement(stmt, local_vars)
  File "/usr/local/bin/youtube-dl/youtube_dl/jsinterp.py", line 55, in interpret_statement
    v = self.interpret_expression(expr, local_vars, allow_recursion)
  File "/usr/local/bin/youtube-dl/youtube_dl/jsinterp.py", line 135, in interpret_expression
    self._objects[variable] = self.extract_object(variable)
  File "/usr/local/bin/youtube-dl/youtube_dl/jsinterp.py", line 220, in extract_object
    fields = obj_m.group('fields')
AttributeError: 'NoneType' object has no attribute 'group'
 (caused by AttributeError("'NoneType' object has no attribute 'group'",)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 828, in _decrypt_signature
    video_id, player_url, s
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 755, in _extract_signature_function
    cache_res = res(test_string)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 807, in <lambda>
    return lambda s: initial_function([s])
  File "/usr/local/bin/youtube-dl/youtube_dl/jsinterp.py", line 254, in resf
    res, abort = self.interpret_statement(stmt, local_vars)
  File "/usr/local/bin/youtube-dl/youtube_dl/jsinterp.py", line 55, in interpret_statement
    v = self.interpret_expression(expr, local_vars, allow_recursion)
  File "/usr/local/bin/youtube-dl/youtube_dl/jsinterp.py", line 135, in interpret_expression
    self._objects[variable] = self.extract_object(variable)
  File "/usr/local/bin/youtube-dl/youtube_dl/jsinterp.py", line 220, in extract_object
    fields = obj_m.group('fields')
AttributeError: 'NoneType' object has no attribute 'group'
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 661, in extract_info
    ie_result = ie.extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 290, in extract
    return self._real_extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 1367, in _real_extract
    encrypted_sig, video_id, player_url, age_gate)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 838, in _decrypt_signature
    'Signature extraction failed: ' + tb, cause=e)
ExtractorError: Signature extraction failed: Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 828, in _decrypt_signature
    video_id, player_url, s
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 755, in _extract_signature_function
    cache_res = res(test_string)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/youtube.py", line 807, in <lambda>
    return lambda s: initial_function([s])
  File "/usr/local/bin/youtube-dl/youtube_dl/jsinterp.py", line 254, in resf
    res, abort = self.interpret_statement(stmt, local_vars)
  File "/usr/local/bin/youtube-dl/youtube_dl/jsinterp.py", line 55, in interpret_statement
    v = self.interpret_expression(expr, local_vars, allow_recursion)
  File "/usr/local/bin/youtube-dl/youtube_dl/jsinterp.py", line 135, in interpret_expression
    self._objects[variable] = self.extract_object(variable)
  File "/usr/local/bin/youtube-dl/youtube_dl/jsinterp.py", line 220, in extract_object
    fields = obj_m.group('fields')
AttributeError: 'NoneType' object has no attribute 'group'
 (caused by AttributeError("'NoneType' object has no attribute 'group'",)); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

**Comment by sandalle** [comment]
This is fixed with 2015.11.24, looks like via 4c7d816dd70a7b10dc096d815da55d41c46f9b59 .

**Comment by yan12125** [comment]
@bubbleguuum Tunein ```/topic/``` URLs are not supported yet.
@sandalle The problem you've encountered is about YouTube. They're different.
